tests/test-team/pages/template-mgmt-base-page.ts

```diff
@@ -13,24 +13,34 @@ export class TemplateMgmtBasePage {
 
   readonly continueButton: Locator;
 
+  readonly skipLink: Locator;
+
   constructor(page: Page) {
     this.page = page;
+
     this.notifyBannerLink = page.locator(
       '[class="nhsuk-header__link nhsuk-header__link--service"]'
     );
+
     this.loginLink = page
       .locator('[class="nhsuk-account__login--link"]')
       .and(page.getByText('Log in'));
+
+    // Note: doing [class="nhsuk-back-link__link"] will not find the element if it has other class names
     this.goBackLink = page
-      .locator('[class="nhsuk-back-link__link"]')
+      .locator('.nhsuk-back-link__link')
       .and(page.getByText('Go back'));
-    this.pageHeader = page
-      .locator('[class="nhsuk-heading-xl"]')
-      .and(page.locator('h1'));
+
+    this.pageHeader = page.locator('h1');
```

Comment:
this changed because not all page headers have the `nhsuk-heading-xl`.

Reply:
👍